### PR TITLE
Add to Anime.md, Manga.md, and Misc.md

### PR DESCRIPTION
**Pull request by Bulldog3321**
Anime.md: AniRena and update some links

Manga.md: MangaReader

Misc.md;
Browser Extensions: Allow Right Click, Return YouTube Dislike, Media Reaper
Browser UserScripts: Direct links out, Microsoft Store Direct Download
Useful Apps: NanaZip

### CODE REVIEW

File: Utilities/Misc.md [hunk 1]
```diff
@@ -46,10 +48,16 @@ description: Tools and Apps to make your life easier.
...
 
+[**Allow Right Click**](https://add0n.com/allow-right-click.html) - Re-enable right clicking on sites that disable it.
+
 [**SponsorBlock**](https://sponsor.ajay.app/) - Skip YouTube Sponsorships & Shoutouts.
 
+[**Return YouTube Dislike**](https://returnyoutubedislike.com/) - Unhides the dislike count on Youtube.
+
 [**DownThemAll!**](https://www.downthemall.org/) | [**DownZemAll!**](https://setvisible.github.io/DownZemAll/) - DownThemAll is a mass download manager directly built inside the Browser, whereas DownZemAll is a fork of DownThemAll with more features but a required desktop client for it to function. They are mostly used for site scraping (such as images).
 
+[**Media Reaper (<span style="color:red">Firefox</span>)**](https://addons.mozilla.org/en-US/firefox/addon/media-reaper/) - Easily download videos and audio files from any site that youtube-dl supports.
```
Comment (Helmasko) [inline Utilities/Misc.md:59]: this seems to support chromium https://sasrip.sas41.com/Home/MediaReaper

File: Utilities/Misc.md [hunk 2]
```diff
@@ -73,9 +83,11 @@ description: Tools and Apps to make your life easier.
 
 [**Direct Download from Google Play**](https://greasyfork.org/en/scripts/33005-direct-download-from-google-play/) - Adds APK-DL, APKPure, APKCombo, APKPremier, APKMirror and Evozi download buttons to Google Play Store when browsing apps, allowing the download of the APK.
 
+[**Microsoft Store Direct Download**](https://greasyfork.org/en/scripts/394420-microsoft-store-direct-download) - Adds DDL links to Microsoft store web pages.
```
Comment (Helmasko) [inline Utilities/Misc.md:86]: i couldnt get it to work, can you confirm this work?

File: Utilities/Misc.md [hunk 3]
```diff
@@ -46,10 +48,16 @@ description: Tools and Apps to make your life easier.
 [**Temporary Containers (<span style="color:red">Firefox</span>)**](https://github.com/stoically/temporary-containers) - Allows you to open tabs, websites, and links in automatically managed disposable containers. Containers isolate data websites store (cookies, storage, and more) from each other.  
 [**Cookie AutoDelete**](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete) - Deletes cookies and other browsing site data as soon as a Browser Tab closes, domain changes or browser restarts.
 
+[**Allow Right Click**](https://add0n.com/allow-right-click.html) - Re-enable right clicking on sites that disable it.
+
 [**SponsorBlock**](https://sponsor.ajay.app/) - Skip YouTube Sponsorships & Shoutouts.
 
+[**Return YouTube Dislike**](https://returnyoutubedislike.com/) - Unhides the dislike count on Youtube.
```
Comment (Helmasko) [inline Utilities/Misc.md:55]: im slightly unsure if this fits the purpose of our guide though we have included youtube age bypass so...
@kekkodance @diedoesmc opinions?

**Review by Helmasko — COMMENTED** [review]
just testing github review feature

File: Utilities/Misc.md (hunk 3 above)
Comment (kekkodance) [reply Utilities/Misc.md:55]: good

File: Utilities/Misc.md (hunk 1 above)
Comment (Bulldog3321) [reply Utilities/Misc.md:57]: right, forgot about that

File: Utilities/Misc.md (hunk 2 above)
Comment (Bulldog3321) [reply Utilities/Misc.md:84]: yea, doesnt work. ms store got redesigned apparently

**Comment by Helmasko** [comment]
Thanks for the contribution doggyBull!

**Pull request merged**